rusk/Cargo.toml

```diff
@@ -99,15 +99,14 @@ rustc_tools_util = { workspace = true }
 default = [
     "ephemeral",
     "recovery-state",
-    "recovery-keys",
     "prover",
     "chain",
     "http-wasm",
 ]
-ephemeral = ["dep:rusk-recovery", "dep:tempfile", "recovery-state", "chain"]
+ephemeral = ["recovery-state", "chain"]
```

Comment:
Are you sure we can remove `dep:tempfile` here? It's still an optional dependency and used in the `ephemeral` feature.

Reply:
`ephemeral` uses `recovery-state` that already uses `dep:tempfile`

It was a leftover duplication

Reply:
> `ephemeral` uses `recovery-state` that already uses `dep:tempfile`
> 
> It was a leftover duplication

Ah yes, I missed that. Thanks!